docs/manual/cli/user.en.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "contao:user"
+description: "Create and list Contao users."
+aliases:
+    - /en/cli/user/
+---
+
+
+After the [database](/en/cli/migrate/) has been created, Contao [users](/en/user-management/users/) can be created here. You can get 
+an overview of the commands via:
+
+```bash
+php vendor/bin/contao-console contao:user --help
+```
```

Comment:
Such a command does not actually exist - i.e. there is no actual `contao:user` command.

Reply:
But lists me the available commands for the "contao:user" namespace.

Reply:
> For the contao:user:password command I would also mention the required username argument.

Changed

Reply:
> For the contao:user:create command I would also mention, that it will interactively ask you for all the user's details when executed without any options.

Added

Reply:
> But lists me the available commands for the "contao:user" namespace

It shows an error that the command does not exist.

Reply:
Deleted.